code/drasil-docLang/Drasil/Sections/SpecificSystemDescription.hs

```diff
@@ -244,7 +246,7 @@ inDataConstTbl qlst = Table titl cts (S "Input Data Constraints") True "InDataCo
             (titleize' physicalConstraint, map fmtPhys qlst),
             (titleize' softwareConstraint, map fmtSfwr qlst),
             (S "Typical Value", map (\q -> fmtU (E $ getRVal q) q) qlst),
-            (short typUnc, map typUncr qlst)]
+            (S "Uncert.", map typUncr qlst)]
```

Comment:
This is not necessarily a good change. We don't want to hard-code something that could be used in multiple sections.

Reply:
`Uncert.` should be the abbreviation, but we want something similar to the `typUnc` chunk that you removed. It should be defined somewhere in Data.Drasil so it can be imported and used where necessary.